pages/common/git-authors.md

```diff
@@ -1,17 +1,17 @@
 # git authors
 
-> Display a log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
+> Display a list of committers of a Git repository. Everything except `--list` will append the list of commiters to the `AUTHORS` file.
```

Comment:
Even tough it does append it, I'm not sure if that's an important detail, since in almost all cases it will create it.

Reply:
If you have a header in the `AUTHORS` file, I think you don't want to recreate a get a mess with the list.

Reply:
Let's move this note to the `--list` example.
```suggestion
> Generate a list of committers of a Git repository.
```
"Generate" because this isn't actually reading it out of some secret existing file.  It's generating it based on the commits.

Reply:
Generally speaking I think this detail might be best served in the example description. If we move `--list` to be first, then put this in the description of the 2nd, then reword the first to imply it's similar to the 2nd but is slightly different.